addons/binding/org.openhab.binding.loxone/README.md

```diff
@@ -0,0 +1,125 @@
+# Loxone Binding
```

Comment:
Thank your for the examples. One more thing that I noticed while reading the `README`: We do have examples now for `.items`, `.things` and `.sitemap` in there but they are hard to find. Could you maybe split the explaining text about those sections from the example code and maybe format the example similar to the [Philipps Hue README](https://github.com/eclipse/smarthome/blob/master/extensions/binding/org.eclipse.smarthome.binding.hue/README.md#full-example)?